nautobot/utilities/forms/fields.py

```diff
@@ -7,18 +7,18 @@
 from django import forms
 from django.apps import apps
 from django.contrib.contenttypes.models import ContentType
-from django.forms.fields import JSONField as _JSONField, InvalidJSONInput
+from django.contrib.postgres.forms import SimpleArrayField
 from django.core.exceptions import MultipleObjectsReturned, ObjectDoesNotExist, ValidationError
 from django.db.models import Count
-from django.forms import BoundField
+from django.forms.fields import BoundField, JSONField as _JSONField, InvalidJSONInput
```

Comment:
Doesn't look like these new imports are used?

Reply:
They are used in in the custom overload of `JSONField`? Just leaving it there for apples-to-apples.

https://github.com/nautobot/nautobot/blob/8d77c1d6842e4f10ea3c190b581317c79f51368a/nautobot/utilities/forms/fields.py#L486-L502

Reply:
On that note, it looks like there's mixed use of THIS `nautobot.utilties.forms.fields.JSONField` and the built-in one from `django.db.models.JSONField`. 🙈